interface/src/app/gene/gene-network.tpl.html

```diff
@@ -2,17 +2,28 @@
   <h1 class="page-header">
     Gene Network
   </h1>
-  <autocomplete-search-panel organism="searchCtrl.organism" autocomplete="searchCtrl.autocomplete"
-    ng-if="searchCtrl.autocomplete">
+
+  <m-l-model-selector selected-m-l-model="searchCtrl.selectedMLModel">
```

Comment:
I just feel the `m-l-` part of the tag a little bizarre. If it is acceptable to the other reviewers, I am fine with it.